### PR TITLE
feat: changed events to use BC specific errorCodes instead of errorInformation messages

### PR DESCRIPTION
**Pull request by ruirochawork**

### CODE REVIEW

File: packages/public-messages-lib/src/cert-management-bc/requests.ts [hunk 1]
```diff
@@ -85,6 +85,7 @@ export class CertListQueryReceivedEvt extends DomainEventMsg {
     }
 
     validatePayload(): void {
+        // NOT IMPLEMENTED
```
Comment (elnyry-sam-k) [inline packages/public-messages-lib/src/cert-management-bc/requests.ts:88]:
```suggestion
        // TODO: NOT IMPLEMENTED
```

File: packages/public-messages-lib/src/cert-management-bc/responses.ts [hunk 2]
```diff
@@ -61,6 +61,7 @@ export class CertListQueryResponseEvt extends DomainEventMsg {
     }
 
     validatePayload(): void {
+        // NOT IMPLEMENTED
```
Comment (elnyry-sam-k) [inline packages/public-messages-lib/src/cert-management-bc/responses.ts:64]:
```suggestion
        // TODO: NOT IMPLEMENTED
```

File: packages/public-messages-lib/src/cert-management-bc/responses.ts [hunk 3]
```diff
@@ -85,6 +86,7 @@ export class CertQueryResponseEvt extends DomainEventMsg {
...
```
Comment (elnyry-sam-k) [inline packages/public-messages-lib/src/cert-management-bc/responses.ts:89]:
```suggestion
        // TODO: NOT IMPLEMENTED
```

**Comment by elnyry-sam-k** [comment]
Thanks @ruirochawork , having todo tag will help catch it in sonarcloud analysis so that it can be addressed at some point..

**Comment by ruirochawork** [comment]
> Thanks @ruirochawork , having todo tag will help catch it in sonarcloud analysis so that it can be addressed at some point..

Done. Thanks for pointing that out @elnyry-sam-k

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**